### PR TITLE
Translation fix

### PR DESCRIPTION
**Pull request by nicobubulle**
Another translation fix.

### CODE REVIEW

**Comment by kberg** [comment]
Lint is complaining: https://github.com/terraforming-mars/terraforming-mars/actions/runs/3461566578/jobs/5779337004#step:5:66

File: src/client/components/Awards.vue [hunk 1]
```diff
@@ -2,16 +2,7 @@
...
-          @click.prevent="toggleList"
-          class="ma-clickable awards-padding"
-          href="#"
-          v-i18n
-          data-test="toggle-awards"
-        >
-          Awards
-        </a>
-
+        <a @click.prevent="toggleList" class="ma-clickable awards-padding" href="#" data-test="toggle-awards" v-i18n>Awards</a>
```
Comment (kberg) [inline src/client/components/Awards.vue:5]: This isn't actually a change, is it?

File: src/client/components/colonies/Colony.vue [hunk 2]
```diff
@@ -86,30 +82,22 @@
...
-      </span>
-      <span v-if="colony.name === ColonyName.HYGIEA" class="colony-background-color" style="margin-left: 3px;">
-        Trade Income: Steal 3 indicated resources
-      </span>
-      <span v-if="colony.name === ColonyName.LEAVITT" class="colony-background-color" style="margin-left: 3px;">
-        Trade Income: Draw X cards and keep 1
-      </span>
+      <span v-if="colony.name !== ColonyName.EUROPA && colony.name !== ColonyName.PLUTO && colony.name !== ColonyName.MERCURY && colony.name !== ColonyName.IAPETUS && colony.name !== ColonyName.HYGIEA && colony.name !== ColonyName.TITANIA && colony.name !== ColonyName.LEAVITT && colony.name !== ColonyName.PALLAS" class="colony-background-color"
+      v-i18n>Trade Income</span>
+      <span v-if="colony.name === ColonyName.PLUTO" class="colony-background-color" style="position:relative; top:-3px"
```
Comment (kberg) [inline src/client/components/colonies/Colony.vue:87]: I appreciate the reformatting, but it makes reading the meaningful differences tricky. If all you did is add v-i18n in these places, can you just keep it as it was?

**Review by kberg — CHANGES_REQUESTED** [review]

File: src/client/components/colonies/Colony.vue (hunk 2 above)
Comment (nicobubulle) [reply src/client/components/colonies/Colony.vue:87]: I noticed that if the string is not inside the tag, the translation is not parsed :
v-i18n>
Trade Income
</span>
Don't work but
v-i18n>Trade Income</span>
works

File: src/client/components/Awards.vue (hunk 1 above)
Comment (nicobubulle) [reply src/client/components/Awards.vue:5]: same issue here

**Review by kberg — APPROVED** [review]

**Pull request merged**